### ISSUE TITLE
Unable to Reproduce Testing Results

### ISSUE THREAD

**Issue by adthorson** [HeewonChung92/CNN_LSTM_HeartRateEstimation#3]
Heewon Chung,

To my understanding, I have implemented your NN architecture using keras 2.2.4.  I was able to reach and exceed your training accuracy; however, I'm unable to reproduce your testing results (BAMI-2).

It would be greatly appreciated if you provided the source code.  I would like to verify if I am somehow incorrectly implementing your NN architecture.

A few questions as well:
- How do you choose the HR value from the222-output layer? Do you choose the associated HR value with the max "FFT" bin / top layer node?
- How do you label your training frames? Average HR within the 8 s window?
- Do you do any dataset leveling?

**Comment by adthorson** [comment]
I have reached out to Prof. Jinseok Lee, who I believe to be a part of your BAMI research lab; however, he has yet to respond.
Thank you!

**Comment by HeewonChung92** [comment]
Hello, Thank you for your interest.

I can't give you all the code, but I can give you the code for the models.
If you write your email address, I will send you a zip file. (related my models)

1. The 222-output layer meanings:
It is the frequency about HR.
It has the 'myFrequency' in my_data.mat files.
and the variable myHR means the HR.
The HR can be found by multiplying the frequency by 60. (myHR = myFrequency * 60).

2. Parameter
input_dim = 222
input_rows = 2
output_dim = 222
time_steps = 6
batch_size = 1

total_epochs = 50
learning_rate = 0.0001
dropRate = 0.3

Have good luck for a new year 😃

**Comment by adthorson** [comment]
Thank you for responding!

My email address is adthorson@wisc.edu

**Comment by mintisan** [comment]
@HeewonChung92 
Could you send a copy to me ?
My email address is mint@mail.ustc.edu.cn.
Thanks.
